### PR TITLE
Add standard mode for linky

### PR DESCRIPTION
**Pull request by Thyphus**

### CODE REVIEW

**Review by fmartinou — APPROVED** [review]
LGTM 👍 

To be merged to feature/#18 then add code quality rules, etc.

File: app/teleinfo/index.js [hunk 1]
```diff
@@ -14,6 +14,430 @@ function isSameFrame(frame1, frame2) {
...
+        {
+           return 1;
+        }
+        else
+        {
+           return -1;
+        }
+    }
+    case 'NGTF':
+    case 'LTARF':
```
Comment (fmartinou) [inline app/teleinfo/index.js:56]: Hi @Thyphus 
It seems that `LTARF` label is present twice in the `switch`.
The 1st time with a control on the value to be shorter than `16 chars` and the 2nd time with a control on the value to be shorter than `9 chars`.

Which one is correct?
(Currently, only the 1st one (< 16 chars) is  used at runtime)

File: app/teleinfo/index.js [hunk 2]
```diff
@@ -14,6 +14,430 @@ function isSameFrame(frame1, frame2) {
...
+    case 'EASD01':
+    case 'EASD02':
+    case 'EASD03':
+    case 'EASD04':
+    case 'EAIT':
+    case 'ERQ1':
+    case 'ERQ2':
+    case 'ERQ3':
+    case 'ERQ4':
+    case 'LTARF':
```
Comment (fmartinou) [inline app/teleinfo/index.js:87]: The 2nd `LTARF` label case.

File: app/teleinfo/index.js [hunk 3]
```diff
@@ -14,6 +14,430 @@ function isSameFrame(frame1, frame2) {
...
+        {
+           return -1;
+        }
+    }
+    case 'PREF':
+    case 'PCOUP':
+    {
+        if(lineItems.length >= 2 && lineItems[1].length == 2)
+        {
+           return 1;
```
Comment (fmartinou) [inline app/teleinfo/index.js:119]: I can see that all this function (regardless for history or stadnard mode) is dealing with returning `-1` and `1`.
And afterwards,
- if the returning value is `-1` you discard the data (considering it invalid I guess) 
- if the returning value is `1` you process the data (considering it valid I guess)

Can't we just rework these functions to basically:
- Return nothing if everything is ok
- throw an error when a corrupted data is detected.

File: app/teleinfo/index.js (hunk 1 above)
Comment (Thyphus) [reply app/teleinfo/index.js:56]: Hi, it's 16 chars.

**Pull request merged**